Features/dko/tile table header menus and sort

DKolar7 opened the pull request.

DKolar7 committed:
Table tile: Enable sort and header menus

Previously disabled sorting possibilities and
header menus are now enabled and available to
use in the Application.

298941

egloff commented on eclipse-scout-core/src/tile/fields/tablefield/TileTableField.js:
```diff
@@ -43,10 +43,7 @@ export default class TileTableField extends TableField {
-      this.table.setHeaderMenusEnabled(false);
-      // Disable sorting for user but don't disabled sorting in Java because CoreTableTile needs sorting functionality.
-      this.table.setSortEnabled(false);
+      this.table.setSortEnabled(true);
```
Is this still needed? I think the default value is true...

DKolar7 committed:
Table tile: Remove obsolete code

Since sorting is enabled by default,
the manually enabling can be removed.

298941

egloff commented:
I squashed the two commits and created a new pull-request: #69
